core/transport.js

```diff
@@ -92,12 +94,16 @@ module.exports = function (driver) {
 
   function setMu (muInstance) {
     mu = muInstance
+    logger = mu.log.child({ muid })
```

Comment:
this isn't nice - something about the way we're initalizing mu is sending the wrong developer pattern signals